app/views/admin/legislation/homepages/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= render "admin/shared/globalize_locales", resource: @process  %>
```

Comment:
We've recently added another partial which doesn't display the options to add and remove languages (that should be done from the "Information" section; otherwise adding new languages will result in validation errors). If you rebase this branch agasint latest master, we could replace this code with:

```erb
<%= render "admin/shared/globalize_tabs",
           resource: @process,
           display_style: lambda { |locale| enable_translation_style(@process, locale) } %>
```
Not very pretty, I know :sweat_smile: ; we've just added the new partial, so it can certainly be improved.

If you aren't familiar with `git rebase`, feel free to leave this line as it is now, and we'll apply the suggested change later :wink:.

Reply:
ok, thanks